mesonbuild/scripts/env2mfile.py

```diff
@@ -421,12 +417,10 @@ def detect_missing_native_binaries(infos: MachineInfo) -> None:
             infos.binaries[toolname] = [exe]
 
 def detect_native_env(options: T.Any) -> MachineInfo:
-    use_for_build = has_for_build()
-    if use_for_build:
-        mlog.log('Using FOR_BUILD envvars for detection')
+    if options.use_for_build:
+        mlog.log('Using FOR_BUILD envvars for detection (native file for use during cross compilation)')
```

Comment:
The help text uses `_FOR_BUILD`, this log message (and the commit message) use `FOR_BUILD`. I would use `_FOR_BUILD` everywhere, but maybe `_FOR_BUILD suffixed envvars` or `*_FOR_BUILD envvars` would be even better.

Reply:
Fixed, thanks.